databases/week1/homework.sql

```diff
@@ -0,0 +1,85 @@
+USE Anton;
+SELECT
+COUNT(*)
+FROM
+task;
+SELECT
+COUNT(title)
+FROM
+task
+WHERE
+due_date IS NULl;
+
+SELECT
+task.title,
+status.name
+FROM
+task,
+status
+WHERE
+task.status_id = status.id
+AND status.name = "Anton";
+SELECT
+task.title,
+status.name
+FROM
+task
+JOIN status ON task.status_id = status.id
+WHERE
+task.status_id = status.id
+AND status.name = "Anton"
```

Comment:
Hmm not sure what you're trying to do here, there's no status with a status name of "Anton"